src/Nethermind/Nethermind.Cli/NodeManager.cs

```diff
@@ -89,6 +89,10 @@ public async Task<JsValue> PostJint(string method, params object[] parameters)
                     decimal totalMicroseconds = stopwatch.ElapsedTicks * (1_000_000m / Stopwatch.Frequency);
                     Colorful.Console.WriteLine($"Request complete in {totalMicroseconds}μs");
                     string? resultString = result?.ToString();
+                    if (result is bool)
+                    {
+                        resultString = resultString?.ToLower();
+                    }
```

Comment:
move it before `result?.ToString()`,  instead of using ToLower, just hardcode `"false"`  and `"true"`  - no allocations.